From a9e8f42603d18af3ad18bc8f16f7d443e8fcafa1 Mon Sep 17 00:00:00 2001
From: ImFatF1sh <zijianjiao2017@gmail.com>
Date: Fri, 26 Mar 2021 17:08:10 +0800
Subject: [PATCH] fixup! dandelion: Add platform privapp permissions

* This will fix if device is stucking on the bootanimation forever

Signed-off-by: ImFatF1sh <zijianjiao2017@gmail.com>
Change-Id: Iddbe33b3c30edc23bd9adbf2f31c379ce022cf94
---
 data/etc/privapp-permissions-platform.xml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/etc/privapp-permissions-platform.xml b/data/etc/privapp-permissions-platform.xml
index 54c548a7b17..0882c3bbced 100644
--- a/data/etc/privapp-permissions-platform.xml
+++ b/data/etc/privapp-permissions-platform.xml
@@ -348,4 +348,8 @@ applications that come with the platform
         <permission name="android.permission.MANAGE_DYNAMIC_SYSTEM"/>
     </privapp-permissions>
 
+    <privapp-permissions package="org.lineageos.xiaomi.imsinit">
+        <permission name="android.permission.INTERACT_ACROSS_USERS"/>
+    </privapp-permissions>
+
 </permissions>
-- 
2.25.1

